Apps/phgsgmail/gsgmail.json

```diff
@@ -13,7 +13,7 @@
     "min_phantom_version": "4.10.0.40961",
     "python_version": "3",
     "latest_tested_versions": [
-        "Cloud, June 7, 2021"
+        "Cloud, September 27, 2021"
     ],
     "app_version": "2.2.0",
```

Comment:
same as the previously released version. please +1 minor version as per standard practice.